src/Microsoft.ML.CodeGen/Templates/Console/ModelInputClass.cs

```diff
@@ -47,13 +44,35 @@ namespace  ");
 
 public IList<string> ClassLabels {get;set;}
 public string Namespace {get;set;}
+public CSharp.CodeGeneratorSettings.GenerateTarget Target {get;set;}
+
+
+void CLI_Annotation()
+{
+this.Write(@"//*****************************************************************************************
+//*                                                                                       *
+//* This is an auto-generated file by Microsoft ML.NET CLI (Command-Line Interface) tool. *
```

Comment:
Looks to me this needs to be moved out into a function, somehting like AddAnnotation().  That uses correct wording :)

Reply:
this is auto-generated by text template `ModelInputClass.tt`, so no need to worry about